### PR TITLE
Remove spaces and change to trailing closure

### PR DESCRIPTION
**Pull request by ambas**
# Changes in this pull request

- remove some extra spaces
- ~~change `updateAccumulatingResult` parameter for `reduce` method to  trailing closure style.~~

#

- [x] All tests pass.

### CODE REVIEW

File: URLQueryItemEncoder/URLQueryItemEncoder.swift [hunk 1]
```diff
@@ -60,9 +59,9 @@ extension Array where Element == CodingKey {
     guard !isEmpty else { return "" }
     var keysPath = self
     let firstKey = keysPath.removeFirst()
-    let tailCodingKeyString = keysPath.reduce(into: "", {
+    let tailCodingKeyString = keysPath.reduce(into: "") {
       $0 += "[\($1.stringValue)]"
-    })
```
Comment (pitiphong-p) [inline URLQueryItemEncoder/URLQueryItemEncoder.swift]: My personal style is to use the trailing closure syntax for the control flow code. So I prefer the normal function call syntax here

File: URLQueryItemEncoder/URLQueryItemEncoder.swift [hunk 2]
```diff
@@ -451,6 +448,3 @@ fileprivate class UnkeyedURLQueryItemReferencingEncoder: URLQueryItemReferencing
     referencedUnkeyedContainer.encodedItemsCount += items.count
   }
 }
-
-
-
```
Comment (pitiphong-p) [inline URLQueryItemEncoder/URLQueryItemEncoder.swift]: Please leave 1 or 2 empty line here. Just an a coding style of an old Objective-C :P

**Review by pitiphong-p — CHANGES_REQUESTED** [review]

File: URLQueryItemEncoder/URLQueryItemEncoder.swift (hunk 2 above)
Comment (ambas) [reply URLQueryItemEncoder/URLQueryItemEncoder.swift]: but as I understand in Xcode 9 there is an extra space in the end of file already, do we need to add more extra.

anyway, I just put an empty space at the end of file so you can see new line in github PR 😬 .

File: URLQueryItemEncoder/URLQueryItemEncoder.swift [hunk 3]
```diff
@@ -63,7 +62,7 @@ extension Array where Element == CodingKey {
     let tailCodingKeyString = keysPath.reduce(into: "", {
       $0 += "[\($1.stringValue)]"
     })
-
```
Comment (pitiphong-p) [inline URLQueryItemEncoder/URLQueryItemEncoder.swift]: One more request, could you put spaces back here?

File: URLQueryItemEncoder/URLQueryItemEncoder.swift (hunk 1 above)
Comment (ambas) [reply URLQueryItemEncoder/URLQueryItemEncoder.swift]: 🙇

File: URLQueryItemEncoder/URLQueryItemEncoder.swift (hunk 3 above)
Comment (ambas) [reply URLQueryItemEncoder/URLQueryItemEncoder.swift]: fixing now....

**Review by pitiphong-p — APPROVED** [review]

**Comment by pitiphong-p** [comment]
Thanks 👍

**Comment by ambas** [comment]
🙇

**Pull request merged**